Detectors/TPC/workflow/src/ZSSpec.cxx

```diff
@@ -0,0 +1,265 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "TPCWorkflow/CATrackerSpec.h"
+#include "TPCWorkflow/ZSSpec.h"
+#include "Headers/DataHeader.h"
+#include "Framework/WorkflowSpec.h" // o2::framework::mergeInputs
+#include "Framework/DataRefUtils.h"
+#include "Framework/DataSpecUtils.h"
+#include "Framework/ControlService.h"
+#include "Framework/ConfigParamRegistry.h"
+#include "DataFormatsTPC/TPCSectorHeader.h"
+#include "DataFormatsTPC/ZeroSuppression.h"
+#include "DataFormatsTPC/ClusterGroupAttribute.h"
+#include "DataFormatsTPC/ClusterNative.h"
```

Comment:
Are you sure all these headers are needed for the ZSSpec? E.g. why would it need ClusterNative.h?